gulpfile.js

```diff
@@ -139,6 +140,7 @@ function initWatch() {
 
     gulp.watch("./pxtpy/**/*", gulp.series(pxtpy, ...tasks.slice(3)));
     gulp.watch("./pxtblocks/**/*", gulp.series(pxtblocks, ...tasks.slice(4)));
+    gulp.watch("./pxtservices/**/*", gulp.series(pxtservices, ...tasks.slice(4)));
```

Comment:
Does the task index need to increment?
```suggestion
    gulp.watch("./pxtservices/**/*", gulp.series(pxtservices, ...tasks.slice(5)));
```

Reply:
nope, it's the same because pxtblocks and pxtservices run at the same step